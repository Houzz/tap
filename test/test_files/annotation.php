<?php

/**
 * @param string $str
 * @param int|null $arg2
 * @return string|bool
 */
function check($str, $arg2) {
    $result = $str . "cd";
    return $result;
}

function callerCorrect() {
    check("abc", 2);
}

function callerCorrectFewer() {
    check("abc");
}

function callerWrongTooFew() {
    check();
}

function callerWrongTooMany() {
    check("abc", 2, 3);
}

/**
 * @param int[]|string[] $a
 */
function testArray($a) {
}

/**
 * @param $a
 */
function testVarNameOnly($a) {
    __tap_check($a, "any");
}

