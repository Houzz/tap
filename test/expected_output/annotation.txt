22 E_FUNCTION_ARGUMENT
26 E_FUNCTION_ARGUMENT
